tests/cpp/test_gather.cpp

```diff
@@ -726,6 +770,8 @@ TEST_F(GatherTest, TakeAlongAxisIntermediateTensorNormalization1) {
   auto tv6 = takeAlongAxis(tv4, tv5, 1);
   fusion.addOutput(tv6);
 
+  at::manual_seed(0);
```

Comment:
Not directly related, but this test exhibited non-deterministic rounding errors. This should make it reproducible.